source/manual/get-started.html.md

```diff
@@ -37,4 +37,9 @@ The easiest way to develop on GOV.UK is to use [govuk-docker][]. Until July 2019
 
 👉 [Setup your AWS account](/manual/set-up-aws-account.html)
 
+## 5. Get GCP access
+
+👉 [Setup your Google Cloud Platform (GCP) access](/manual/set-up-gcp-account.html)
```

Comment:
Given that this access is "granted with full production access", this link belongs in https://docs.publishing.service.gov.uk/manual/rules-for-getting-production-access.html rather than the day 1 Getting Started guide for a new dev.

Reply:
Fixed with last commit